src/storage-client/src/types/connections.rs

```diff
@@ -811,9 +883,9 @@ impl Arbitrary for PostgresConnection {
     }
 }
 
-/// Specifies how to tunnel a [`PostgresConnection`].
+/// Specifies how to tunnel a Connection.
```

Comment:
This isn't a specific code class, right?

```suggestion
/// Specifies how to tunnel a connection.
```